src/app/components/register/registerController.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * File: LoginController.js
+ * Type: ngController
```

Comment:
Change this comment to reflect the correct name of the file as well as the description.